shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/di/koin/modules/ViewModelModule.kt

```diff
@@ -1,8 +1,10 @@
 package co.nimblehq.blisskmmic.di.koin.modules
 
 import co.nimblehq.blisskmmic.presentation.modules.login.LoginViewModel
+import co.nimblehq.blisskmmic.presentation.modules.resetpassword.ResetPasswordViewModel
 import org.koin.dsl.module
 
 val viewModelModule = module {
     single<LoginViewModel> { LoginViewModel(get()) }
+    single { ResetPasswordViewModel(get()) }
```

Comment:
Optional: I think we can apply the newly introduced syntax for `LoginViewModel` module also 😁
```suggestion
    single { LoginViewModel(get()) }
    single { ResetPasswordViewModel(get()) }
```